models/localized-string/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@commercetools-test-data/localized-string",
+  "version": "0.1.0",
+  "description": "Data model for LocalizesString type",
+  "bugs": "https://github.com/commercetools/test-data/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/test-data.git",
+    "directory": "models/localized-string"
+  },
+  "keywords": [
+    "javascript",
+    "test-data"
+  ],
+  "license": "MIT",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./dist/localized-string-index.cjs.js",
+  "module": "./dist/localized-string-index.es.js",
+  "typings": "./dist/typings/index.d.ts",
+  "types": "./dist/typings/index.d.ts",
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "yarn build:bundles && yarn build:typings",
+    "build:bundles": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js -i ./src/index.ts --dir dist",
+    "build:bundles:watch": "yarn build:bundles -w",
+    "build:typings": "cross-env tsc -p tsconfig.declarations.json --emitDeclarationOnly --declarationDir dist/typings"
+  },
+  "dependencies": {
+    "@babel/runtime": "7.11.2",
+    "@babel/runtime-corejs3": "7.11.2",
+    "@commercetools-test-data/core": "1.0.1",
+    "faker": "^4.1.0"
```

Comment:
Shouldn't be fixed?

```suggestion
    "faker": "4.1.0"
```

Reply:
I chose to have it like this to avoid different versions with the `faker` dependency coming from the test-data-bot package.

Maybe we can just re-export it via the core package, that would be better maybe